edx_api_doc_tools/example/views.py

```diff
@@ -0,0 +1,102 @@
+"""
+REST API views for reading and writing to the edX Hedgehog Database.
+
+Documented using edx_api_doc_tools.
+
+TODO finish documenting HedgehogViewSet.
+TODO also give an example of documenting "traditional" (non-ViewSet) DRF views.
+"""
+from __future__ import absolute_import, unicode_literals
+
+from rest_framework.exceptions import APIException, NotFound
+from rest_framework.viewsets import ModelViewSet
+
+from edx_api_doc_tools import path_parameter, query_parameter, schema_for
+
+from .data import get_hedgehogs
+from .serializers import HedgehogSerializer
+
+
+@schema_for(
+    'list',
+    """
+    Fetch the list of edX hedgehogs.
+
+    Hedgehogs can be filtered by minimum weight (grams or ounces),
+    their favorite food, whether they graduated college,
+    or any combination of those criterion.
+    """,
+    parameters=[
+        query_parameter('min-grams', int, "Filter on whether minimum weight (grams)."),
+        query_parameter('min-ounces', float, "Filter hogs by minimum weight (ounces)."),
+        query_parameter('fav-food', str, "Filter hogs by favorite food."),
+        query_parameter('graduated', bool, "Filter hogs by whether they graudated."),
+    ],
+)
+@schema_for(
+    'retrieve',
+    # TODO: The below string becomes the operation's description instead
+    #       of its summary for some reason.
```

Comment:
Something wrong with the multi-line decision making?

Reply:
Yeah, I'm debugging it currently.